python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1041,7 +1043,7 @@ def _combineExposureMetadata(self, visit, dataRefs, isGen3=True):
 
             rec['physical_filter'] = filterLabel.physicalLabel if filterLabel.hasPhysicalLabel() else ""
             rec['band'] = filterLabel.bandLabel if filterLabel.hasBandLabel() else ""
-            rec['detector_id'] = detector.getId()
+            rec['id'] = detector.getId()
```

Comment:
I don't know that it makes any real difference, but you can also do `rec.setId(detector.getId())` here, since every Catalog has `id`.